mmcv/ops/deform_conv.py

```diff
@@ -224,7 +224,8 @@ def __init__(self,
                  dilation: Union[int, Tuple[int, ...]] = 1,
                  groups: int = 1,
                  deform_groups: int = 1,
-                 bias: bool = False) -> None:
+                 bias: bool = False,
+                 im2col_step: int = 32) -> None:
```

Comment:
Add a definition for `im2col_step` in the docstring, and it would be better to provide solved issue link in the definition.